Doc/library/mimetypes.rst

```diff
@@ -272,3 +272,96 @@ than one MIME-type database; it provides an interface similar to the one of the
       types, else to the list of non-standard types.
 
       .. versionadded:: 3.2
+
+
+.. mimetypes-cli:
+
+Command-Line Usage
+------------------
+
+The :mod:`mimetypes` module can be executed as a script from the command line.
+
+.. code-block:: sh
+
+   python -m mimetypes [-e] [-l] type [type ...]
+
+The following options are accepted:
+
+.. program:: mimetypes
+
+.. cmdoption:: -h
+               --help
+
+   Show the help message and exit.
+
+.. cmdoption:: -e
+               --extension
+
+   Guess extension instead of type.
+
+.. cmdoption:: -l
+               --lenient
+
+   Additionally search for some common, but non-standard types.
+
+The script converts file extensions to MIME types if ``--extension`` option
+is specified, or vice versa if not.
+
+For each ``type`` entry, the script writes a line into the standard output
+stream. If an unknown type occurs, it writes an error message into the
+standard error stream and aborts with the return code ``1``.
+
+
+.. mimetypes-cli-example:
+
+Command-Line Example
+--------------------
+
+Here are some examples of typical usage of the :mod:`mimetypes` command
+line interface:
+
+.. code-block:: shell
+
+   # get a MIME type by a file name
+   $ python -m mimetypes filename.png
```

Comment:
Some samples have `$` prefix, some don't. I propose to remove all of them. 

Grep for `$ python`, you won't find many examples.

Reply:
You're right, `Doc/library` in [`main`](https://github.com/python/cpython/tree/e244401ce508ad391295beb636e499fcc6797a2a) has 14 prefixed submodule calls vs 18 non-prefixed.

```bash
$ git grep "$ python -m" | awk -F ':' '{print $1}' | sort -u | uniq | wc -l
14
$ git grep "$ python -m" | awk -F ':' '{print $1}' | sort -u | uniq
__main__.rst
asyncio.rst
importlib.metadata.rst
json.rst
pickletools.rst
shutil.rst
site.rst
sysconfig.rst
tarfile.rst
timeit.rst
tokenize.rst
uuid.rst
zipapp.rst
zipfile.rst
```

```bash
$ git grep "[^$] python -m" | awk -F ':' '{print $1}' | sort -u | uniq | wc -l
18
$ git grep "[^$] python -m" | awk -F ':' '{print $1}' | sort -u | uniq
ast.rst
doctest.rst
ensurepip.rst
http.server.rst
itertools.rst
json.rst
profile.rst
py_compile.rst
pydoc.rst
sqlite3.rst
timeit.rst
tokenize.rst
trace.rst
turtle.rst
unittest.rst
uuid.rst
webbrowser.rst
xmlrpc.server.rst
```

Reply:
As for me, it looks confusing (like some config file). Any ideas on what to do next?

![image](https://user-images.githubusercontent.com/4881073/214266481-b251b441-b753-4262-8487-b509bb9b6b5c.png)

Reply:
A side question: is there a point to use `.. code-block:: shell-session` instead, [as in `site`](https://deploy-preview-93097--python-cpython-preview.netlify.app/library/site.html#command-line-interface)?

Reply:
I don't know :)
This is the first time I see this directive.